src/Components/Functionalities/User/VisitHistory.js

```diff
@@ -30,10 +31,10 @@ export default class VisitHistory extends Component {
 
       
 
-    //   apiClient.perform('post', '/visitHistory', this.state).then(resp => {
-    //     var rowData = resp
-    //     this.state.rowData = rowData
-    //   });
+      apiClient.perform('post', '/GetVisitHistory', this.state ).then( resp => {
+        this.setState({rowData: resp['data']},
+        console.log(resp))
+      });
```

Comment:
Might not want to send the whole state

Reply:
fixed in commit [f5c7c70](https://github.com/49ers-DB/Atlanta-Movie/pull/29/commits/f5c7c70f19db0015eda1a6cbbd6e88568480951c)